docs/platform/dedicated-cpu/getting-started-with-dedicated-cpu/index.md

```diff
@@ -16,7 +16,9 @@ This guide will serve as a brief introduction into what a Dedicated CPU Linode i
 
 ## What is a Dedicated CPU Linode?
 
-Whereas a Standard Linode gives you access to shared virtual CPU cores, a dedicated CPU Linode gives you access to entire physical CPU cores accessible only by your Linode. This eliminates the possibility of CPU steal, or what happens when two virtual CPU processes are scheduled at the same time and one has to wait to complete its process. While a standard plan is a good fit for most use cases, a Dedicated CPU Linode is recommended for a number of workloads related to high and constant CPU processing. A few of these are as follows:
+In contrast with a [Standard Linode](/docs/platform/how-to-choose-a-linode-plan/#2-standard), which gives you access to shared virtual CPU cores, a Dedicated CPU Linode offers entire physical CPU cores that are accessible only by your instance. Because your cores will be isolated to your Linode, no other Linodes can schedule processes on them, so your instance will never have to wait for another process to complete its execution, and your software can run at peak speed and efficiency.
```

Comment:
Removed explicit references to CPU steal to avoid the negative connotations of 'steal'. Moved 'cpu steal' into the keywords for the guide, so this guide will still show up for people searching for that term